content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+
+In addition, social media platforms may filter content related directly or indirectly to abortion in users’ states where abortion is not allowed. Some explanations, articles, documentaries on abortion may be censored just to respect the law… VPN services (secured connection that may be used to hide connection location) will have beautiful days ahead. The account of Abortion Finder, a site that allows women who wish to have an abortion to find a health professional, was briefly suspended from Instagram and then reinstated… Oops[^9].
+
+To improve their brand image, Meta promised to cover the costs of its employees having to travel outside their home state for an abortion, and Google claimed that its employees who live in states where abortion is banned may request to be transferred to other locations. Sadly, moving will be the only simple and safe way to deal with these freedom restrictions for women that are concerned and worried about it.
+
+If tech companies worries sincerely about their users' safety, they need to stop collecting our data now, and to not have it when the warrants arrive. At least they must collect less data, encrypt it, not share it with dubious actors, not force users to authenticate, etc. Of course this will negatively impact their business model but maybe it is time for them to change their retribution model. Society is evolving, they must also evolve if they have consideration for their customers.
+
+#### Smartphones and Apps
+
+Another side of this situation is smartphone apps. Some may have guessed the level of knowledge of social media on their users, but maybe some forgot that apps on their smartphone are collecting a lot of sensible information. Again, some apps may use some tracker from facebook that are hidden in some freeware libraries and frameworks they include in their app to save development time. Facebook and Google are everywhere. Android smartphones are at risk but Apple’s smartphones are also at risk even if less exposed.
```

Comment:
```suggestion
Another side of this situation is smartphone apps. Some may have guessed the level of knowledge of social media on their users, but maybe some forgot that apps on their smartphone are collecting a lot of sensible information. Again, some apps may use some trackers from Facebook that are hidden in some freeware libraries and frameworks they include in their app to save development time. Facebook and Google are everywhere. Android smartphones are at risk but Apple’s smartphones are also at risk even if less exposed.
```

Explained more thoroughly in the French part, but this has yet to be proved regarding Apple supposedly having better privacy practices.

Reply:
Removed the last part.